lib/getBinaryGltf.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+var defaultValue = Cesium.defaultValue;
+var sizeOf = require('image-size');
+var mime = require('mime');
+var mergeBuffers = require('./mergeBuffers');
+var removePipelineExtras = require('./removePipelineExtras');
+
+module.exports = getBinaryGltf;
+
+function getBinaryGltf(gltf, callback) {
+    //Create the special binary buffer from the existing buffers
+    gltf.bufferViews = defaultValue(gltf.bufferViews, {});
+    gltf.buffers = defaultValue(gltf.buffers, {});
+    mergeBuffers(gltf, 'binary_glTF');
+
+    var bufferViews = gltf.bufferViews;
+    var buffers = gltf.buffers;
+    var currentOffset = buffers.binary_glTF.byteLength;
+    var body = buffers.binary_glTF.extras._pipeline.source;
+    var currentBinaryView = 0;
+    //Update object with KHR_binary_glTF properties and add to body and bufferViews
+    function updateBinaryObject(name) {
+        var objects = gltf[name];
+        if (defined(objects)) {
+            for (var objectId in objects) {
+                if (objects.hasOwnProperty(objectId)) {
+                    var object = objects[objectId];
+
+                    //Update object with binary format
+                    object.uri = "data:,";
+                    object.extensions = defaultValue(object.extensions, {});
+                    object.extensions.KHR_binary_glTF = defaultValue(object.extensions.KHR_binary_glTF, {});
+                    var KHR_binary_glTF = object.extensions.KHR_binary_glTF;
+
+                    //Create a bufferView based on the byte length and current offset
+                    var bufferViewKeys = Object.keys(bufferViews);
+                    while (bufferViewKeys.indexOf('binary_bufferView' + currentBinaryView) != -1) {
+                        currentBinaryView++;
+                    }
+
+                    var objectSource = object.extras._pipeline.source;
+                    var bufferViewId = 'binary_bufferView' + currentBinaryView;
+                    KHR_binary_glTF.bufferView = bufferViewId; //Create bufferview
+                    bufferViews[bufferViewId] = {
+                        "buffer": "binary_glTF",
```

Comment:
Drop `"` for properties.  Use `'` for the string values.

I don't think gltf-pipeline is completely consistent here, but this is the standard Cesium.  There's some exceptions, for example, if the entire object is JSON (we do this for CZML examples in Sandcastle).